hal/halx86/acpi/madt.c

```diff
@@ -12,77 +13,113 @@
 /* ACPI_BIOS_ERROR defined in acoutput.h and bugcodes.h */
 #undef ACPI_BIOS_ERROR
 #include <smp.h>
+
+// Note: HalpParseApicTables() is called early, so its DPRINT*() do nothing.
 #define NDEBUG
 #include <debug.h>
 
+#define APIC_MODE_LEGACY 0x10
```

Comment:
Declaring this next to `HALP_APIC_INFO_TABLE` would seem nicer.

Reply:
Moved.